server/apiserver/apiserver.go

```diff
@@ -164,6 +164,59 @@ func (api *ApiServer) CallUpdatePixel(x int, y int, r int, g int, b int, userID
 	}
 }
 
+
+/*
+ * Insert the new user id to the userlist
+ */
+func (api *ApiServer) HTTPUpdateUserList(w http.ResponseWriter, req *http.Request) {
+	// Only for testing
+	user_id := req.URL.Query().Get("user_id")
+	api.CallUpdateUserList(user_id)
+}
+
+/*
+ * Insert the new user id to the userlist
+ */
+func (api *ApiServer) CallUpdateUserList(user_id string) []byte {
+	byt := []byte("")
+	_, ifErr := api.GetLastUserModification(user_id)
+	if (ifErr) {
+		err := api.SetLastUserModification(user_id, "0")
+		if (err) {
+			fmt.Println("Cannot update user list")
+			byt = []byte("Test failure")
+		} else {
+			fmt.Println("Successfully created user")
+			byt = []byte("Successfully created user")
```

Comment:
Like my above comment about the message for failure not having types, all of these strings that are being returned need to be changed into messages/json that can be parsed by the client.

From the design doc (https://docs.google.com/document/d/1KSX_3OYxqOvSgNRE_sHzmTzEveMyIyrT6FSyV5vjZfg/edit#heading=h.s2ff1rxx7nrb) you were thinking of structuring it like 
{
‘type’: “drawPixel”, 
“id”: <user’s Google ID>,
‘status’: <code>
}

where Status Code could be
200: user exists and we can make the move
401: not a valid user
429: user not allowed to make a move at this time

but regardless of the format you need to add these to the messages files.

Reply:
So what is the plan with this?
I see that it is still not changed to have a message type and I see no comment explaining why you did not change it today.

Reply:
This is addressed in the latest commit.